enaml/core/parser/parser36.py

```diff
@@ -32,28 +32,28 @@ class Python36EnamlParser(Python35EnamlParser):
     def p_comp_for1(self, p):
         ''' comp_for : FOR exprlist IN or_test '''
         super(Python36EnamlParser, self).p_comp_for1(p)
-        p[0][0].is_async = False
+        p[0][0].is_async = 0
```

Comment:
Sorry for that the name of the field got me confused. I will propose a patch at https://greentreesnakes.readthedocs.io/en/latest/nodes.html#comprehensions to make it clearer that this uses an int.

Reply:
And this is now merged.